Framework/DataHandling/test/SaveFocusedXYETest.h

```diff
@@ -57,8 +57,10 @@ class SaveFocusedXYETest : public CxxTest::TestSuite {
     std::ifstream filestrm(filename.c_str());
     std::string line;
     int bin_no(1);
+    int lineNumber = 0;
     while (getline(filestrm, line)) {
-      if (line[0] == '#')
+      lineNumber++;
+      if (lineNumber <= 6)
```

Comment:
Can you factor out the 6 as `MAX_HEADER_LENGTH` or similar? `lineNumber < 7` would also read more clearly to me.

Reply:
Good point on having a global variable. But I think setting it to 7 and then asking `<7` would be more confusing, since Fullprof documentation says 6. I think it is better to have 6 and ask `<=6`.

Reply:
Ah, I see, lines are 1 -> 6, with the first line in the file not being part of the header. Seems ok, but you can repeat the `MAX_HEADER_LENGTH` hear and above.